### ISSUE TITLE
If i have Multiple kafka clusters 

### ISSUE THREAD

**Issue by wjw870970** [yahoo/CMAK#735]
One of them is kerberos certified.Others are not. If jaas_conf is referenced globally.How to access those unauthenticated zk？

**Comment by patelh** [comment]
Don't use global config.  Provide jaas config in cluster config.

**Comment by wjw870970** [comment]
Thanks patelh
But kafka-manager.zkhosts connected by CMAK has authentication(kerberos), so how to set it up?

**Comment by patelh** [comment]
ah, that could be a problem.  Let me see how we can resolve it.

**Comment by genzgd** [comment]
We are having a similar problem.  We have multiple Kafka clusters which require SSL/client certificate authentication.  If we add the appropriate keystore and truststore entries in the consumer.properties file, the Kafka ConsumerProperties is correctly created for the first cluster.  However, if we switch clusters, the bootstrap.servers value is still set to the first cluster, so connectivity fails.

In other words, it seems like if you actually set the CONSUMER_PROPERTIES_FILE environment variables (which could be better documented), the KafkaConsumer properties are not entirely cleaned up and replaced when switching clusters.  This is unfortunately a blocking issue for us because it means we can only point CMAK to one SSL secured cluster at a time.

Has then been any progress in this area?

**Comment by wjw870970** [comment]
I chose a zookeeper cluster that does not require authentication for the cmak.And then set authentication configuration for each clusters separately.It should be noted that the content of SASL JAAS Config configuration is not a file
